src/kvstore/NebulaStore.h

```diff
@@ -57,6 +56,7 @@ class NebulaStore : public KVStore, public Handler {
   FRIEND_TEST(NebulaStoreTest, ThreeCopiesCheckpointTest);
   FRIEND_TEST(NebulaStoreTest, RemoveInvalidSpaceTest);
   friend class ListenerBasicTest;
+  friend class TopoListenerTest;
```

Comment:
where it was used?

Reply:
My bad, I'll remove it. It is a legacy code.